manager/api/transport.go

```diff
@@ -281,14 +266,14 @@ func decodeList(_ context.Context, r *http.Request) (interface{}, error) {
 	if len(off) == 1 {
```

Comment:
Maybe you should extract these checks and conversion to int into separate function. It is repeated two times here.